<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionsClass</class>
 <widget class="QDialog" name="ConnectionsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connections</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_in">
     <property name="text">
      <string>MIDI IN</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="m_inputBackends"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="m_inputPorts"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="m_thru">
     <property name="text">
      <string>Enable MIDI THRU</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_out">
     <property name="text">
      <string>MIDI OUT</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="m_outputBackends"/>
   </item>
   <item row="5" column="1">
    <widget class="QToolButton" name="btnOutputDriverCfg">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QComboBox" name="m_outputPorts"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="m_advanced">
     <property name="text">
      <string>Show advanced connections</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="btnInputDriverCfg">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionsClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionsClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
